packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Upgrade to Okta Identity Engine
+title: Upgrade to Identity engine
 meta:
   - name: description
-    content: Our guide shows how to upgrade your organization and clients to OIE 
+    content: Our guide shows how to upgrade your organization and clients to Identity engine
 layout: Guides
 ---
 
-## Enable OIE for your organization
+## Enable Identity engine for your organization
 
-To upgrade to Okta Identity Engine, please reach out to your account manager. If you do not have an account manager, please reach out to <oie@okta.com> for more information.
+To upgrade to Identity engine, please reach out to your account manager. If you do not have an account manager, please reach out to <oie@okta.com> for more information.
```

Comment:
```suggestion
To upgrade to Identity engine, contact your account manager. If you do not have an account manager, email <oie@okta.com> for more information.
```

Reply:
Infodev style doesn't like "please" in this situation. My preference was also to remove the "reach out" for brevity, but just a suggestion.